superface/super.json

```diff
@@ -1331,7 +1331,10 @@
           "id": "apikey",
           "apikey": "$DEEPL_API_KEY"
         }
-      ]
+      ],
+      "parameters": {
+        "SUBDOMAIN": "api-free"
```

Comment:
Is this here to be explicit or the default in parameter definition doesn't work?